.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,40 @@
+<!--
+This issue template is meant mainly for bug reports and feature requests.
+
+You can still use an issue to ask for help. In this case
+you should remove the template contents or use it as a guideline
+for providing information for debugging.
+-->
+
+### Kind of issue
```

Comment:
This section isn't required at all. Feature requests do not follow this template. The contents of feature request can be very arbitrary in nature.

Further, we have labels - `bug` and `feature` that maintainers can set after triaging the issue